Author: Saleem Abdulrasool <compnerd@compnerd.org>
Reason: fix compile failure caused by undefined symbol
Upstream: no
--- 
--- xchat-2.8.6/src/fe-text/fe-text.c	2008-08-23 19:29:47.000000000 -0700
+++ xchat-2.8.6/src/fe-text/fe-text.c	2008-08-23 19:31:12.000000000 -0700
@@ -859,3 +859,4 @@
 void fe_tray_set_icon (feicon icon){}
 void fe_tray_set_tooltip (const char *text){}
 void fe_tray_set_balloon (const char *title, const char *text){}
+void fe_userlist_update (session *sess, struct User *user){}
